app/ui/components/form/input/types/NumberInputAppProps.ts

```diff
@@ -0,0 +1,7 @@
+export declare interface NumberInputAppProps {
```

Comment:
I think its a bad practice to set them all nullable, not sure for this case here tho